producers-commissioning/src/unit-test/java/uk/gov/moj/sdt/producers/comx/sdtws/SdtEndpointPortTypeTest.java

```diff
@@ -73,185 +78,163 @@ public class SdtEndpointPortTypeTest extends AbstractSdtUnitTestBase
-        portType.setWsReadBulkRequestHandler (mockBulkRequestHandler);
+        MockitoAnnotations.openMocks(this);
 
-        mockSubmitQueryHandler = EasyMock.createMock (IWsReadSubmitQueryHandler.class);
-        portType.setWsReadSubmitQueryHandler (mockSubmitQueryHandler);
+        portType = new SdtEndpointPortType();
+        portType.setWsCreateBulkRequestHandler(mockCreateBulkRequestHandler);
+        portType.setWsReadBulkRequestHandler(mockBulkRequestHandler);
+        portType.setWsReadSubmitQueryHandler(mockSubmitQueryHandler);
     }
 
     /**
      * Test submit bulk method completes successfully.
      */
     @Test
-    public void testSubmitBulkSuccess ()
-    {
-        final BulkResponseType dummyResponse = createBulkResponse ();
-        final BulkRequestType dummyRequest = createBulkRequest ();
+    void testSubmitBulkSuccess() {
+        final BulkResponseType dummyResponse = createBulkResponse();
+        final BulkRequestType dummyRequest = createBulkRequest();
 
-        EasyMock.expect (mockCreateBulkRequestHandler.submitBulk (dummyRequest)).andReturn (dummyResponse);
-        EasyMock.replay (mockCreateBulkRequestHandler);
+        when(mockCreateBulkRequestHandler.submitBulk(dummyRequest)).thenReturn(dummyResponse);
 
         final BulkResponseType response = portType.submitBulk (dummyRequest);
 
-        EasyMock.verify (mockCreateBulkRequestHandler);
-        assertNotNull ("Response expected", response);
-        assertEquals (SDT_COMX_SERVICE, response.getSdtService ());
+        verify(mockCreateBulkRequestHandler).submitBulk(dummyRequest);
+        assertNotNull(response, RESPONSE_EXPECTED);
+        assertEquals(SDT_COMX_SERVICE, response.getSdtService());
     }
 
     /**
      * Test submit bulk method handles exceptions successfully.
      */
     @Test
-    public void testSubmitBulkException ()
-    {
-        EasyMock.expect (mockCreateBulkRequestHandler.submitBulk (EasyMock.anyObject (BulkRequestType.class)))
-                .andThrow (new RuntimeException ("test"));
-        EasyMock.replay (mockCreateBulkRequestHandler);
-
-        try
-        {
-            portType.submitBulk (createBulkRequest ());
-            fail ("Runtime exception should have been thrown");
-        }
-        catch (final RuntimeException re)
-        {
-            assertEquals ("",
-                    "A SDT system component error has occurred. Please contact the SDT support team for assistance",
-                    re.getMessage ());
+    void testSubmitBulkException() {
+        when(mockCreateBulkRequestHandler.submitBulk(any(BulkRequestType.class)))
+                .thenThrow(new RuntimeException ("test"));
+
+        try {
+            portType.submitBulk(createBulkRequest());
+            fail(RUNTIME_EXCEPTION_SHOULD_HAVE_BEEN_THROWN);
+        } catch (final RuntimeException re) {
+            assertEquals(SDT_SYSTEM_COMPONENT_ERROR, re.getMessage(), "");

```

Comment:
Doesn't seem to be any benefit in outputting an empty string as the message so might as well use the version of assertEquals() that doesn't specify the message parameter.